client/blocks/reader-full-post/index.jsx

```diff
@@ -119,6 +120,7 @@ export class FullPostView extends React.Component {
 		if ( this.hasCommentAnchor && ! this.hasScrolledToCommentAnchor ) {
 			this.scrollToComments();
 		}
+		iFrameResize();
```

Comment:
The resizer should work in such a way that it attaches the `message` listener on mount, and detaches in on unmount.

To be even more resilient, it should manage only iframes that are children of the `FullPostView` component instance's DOM element. That prevents accidental interaction with unrelated iframes that are in a different part of the Calypso UI.

Instead of querying for `document.querySelectorAll( 'iframe' )`, it should query locally as `this.elRef.current.querySelectorAll( 'iframe' )`.